README.md

```diff
@@ -118,6 +118,18 @@ mvn install
 This will compile DDF and run all of the tests in the DDF source distribution. It usually takes some time for maven to download required dependencies in the first build.
 The distribution will be available under "distribution/ddf/target" directory.
 
+### Run OWASP check
+The OwASP scan for known vulnerabilities is not enabled by default. 
+To scan the artifacts for known vulnerabilities, use the `owasp` profile with the installation command:
+
+```
+mvn install -P owasp
```

Comment:
`package` is the lifecycle phase that we bind plugin execution to, so `mvn package -P owasp` should work just as well (and potentially skip unnecessary plugin/test execution)